src/components/ui/buttons/Button.tsx

```diff
@@ -126,8 +126,9 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
 )
 Button.displayName = "Button"
 
-type ButtonLinkProps = LinkProps &
+type ButtonLinkProps = Omit<LinkProps, "href"> &
   Pick<ButtonProps, "size" | "variant" | "isSecondary"> & {
+    href: string
```

Comment:
just curious, is the intention of this change to make the field required? looks good if so

Reply:
@pettinarip yes. I find it weird that it's not a required thing for the anchor element. 😅